<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2135</width>
    <height>1378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KTrader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ZIP/risk.png</normaloff>../ZIP/risk.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>2041</width>
      <height>1281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>수동</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>651</height>
       </rect>
      </property>
      <property name="title">
       <string>수동주문</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>61</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>124</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>주문</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>201</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>종목</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>334</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>종류</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>412</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>수량</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>502</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>가격</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>58</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>신규매수</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>신규매도</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매수취소</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매도취소</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>지정가</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>시장가</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>193</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>410</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>500</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>570</y>
         <width>150</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>주문실행</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>1611</width>
        <height>651</height>
       </rect>
      </property>
      <property name="title">
       <string>금일 수동매수 종목</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>1561</width>
         <height>391</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>코드명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>매입가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>현재가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>손익액</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>700</y>
        <width>991</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>지수현황</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>700</y>
        <width>971</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>섹터현황</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>자동</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1991</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string>금일 실시간 추천</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>종목명</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>지정가</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>시장가</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>종류</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>320</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>가격</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>255</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>수량</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>370</y>
         <width>150</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>주문</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>1511</width>
         <height>221</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>전략번호</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목번호</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>진입청산</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>추천시간</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>현재가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>진입가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>등락률</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>전략명</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboStg">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>191</width>
         <height>36</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_6">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>270</y>
         <width>1511</width>
         <height>221</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>전략번호</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목번호</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>진입청산</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>추천시간</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>현재가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>진입가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>등락률</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>660</y>
        <width>661</width>
        <height>561</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>가격</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>등락률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>거래량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>거래량증가율</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>660</y>
        <width>651</width>
        <height>561</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>가격</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>등락률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>거래량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>거래량증가율</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>610</y>
        <width>133</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>610</y>
        <width>133</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>계좌</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>1991</width>
        <height>581</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY울릉도B</family>
       </font>
      </property>
      <property name="title">
       <string>잔고 및 보유현황</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>1951</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HY헤드라인M</family>
        </font>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>예수금(d+2)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총매입</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총평가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총손익</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총수익률(%)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>추정자산</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>1951</width>
         <height>341</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HY헤드라인M</family>
        </font>
       </property>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>보유량</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>매입가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>현재가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>평가손익</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률(%)</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>520</y>
         <width>150</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>530</y>
         <width>146</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>실시간</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>검색</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2135</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
